src/api.js

```diff
@@ -174,6 +174,13 @@ export const MOCK_EVENT = {
 	yes_rsvp_count: 23
 };
 
+export const MOCK_BEST_TOPIC = {
```

Comment:
yeah, that might be the best way to go - just nest it in `MOCK_CATEGORY`... if only these things were consistent!!! Turns out there's at least one other 'topic' variant that is returned for member profiles that has `id`, `name`, `urlkey`. Seems like a topic 'type' definition would look like:

```js
{
  id: number,
  name: string,
  urlkey: string,
  lang?: string,
  group_count?: number,
  member_count?: number,
  description?: string,
}
```

`lang` might actually be required now...

maybe just leave it at `MOCK_BEST_TOPIC` 🙂  ... or `MOCK_CATEGORY_BEST_TOPIC`